common/ast/src/parser/expr.rs

```diff
@@ -960,8 +946,8 @@ pub fn expr_element(i: Input) -> IResult<WithSpan<ExprElement>> {
             | #literal : "<literal>"
             | #case : "`CASE ... END`"
             | #subquery : "`(SELECT ...)`"
-            | #any_subquery : "`<op> ANY (SELECT ...)`"
-            | #all_subquery : "`<op> ALL (SELECT ...)`"
+            | #any_op : "`ANY (SELECT ...)`"
+            | #all_op : "`ALL (SELECT ...)`"
```

Comment:
Is it possible to merge these three cases? FYI, `#subquery` already has a `NOT` prefix, where we can add `ANY` and `ALL` variants.